src/main/java/milbot/Ui.java

```diff
@@ -0,0 +1,114 @@
+package milbot;
+
+import taskclasses.*;
+
+/**
+ * Ui class handles user interface-related functionalities.
+ */
+public class Ui {
+    /**
+     * Prints a goodbye message before the chatbot exits.
+     */
+
+    public String printGoodbyeMessage() {
+        return "Have a nice day and see you again soon!";
+    }
+
+    /**
+     * Prints an error message.
+     *
+     * @param message The error message to be printed.
+     */
+
+    public String printErrorMessage(String message) {
+        return message + "\n";
+    }
+
+
+    /**
+     * Prints a message indicating a new task has been added to the task list.
+     *
+     * @param taskList The TaskList containing the added task.
+     * @param task     The added task.
+     */
+
+    public String printNewTask(TaskList taskList, Task task) {
+        String newTaskMessage = "Got it. I've added this task:\n" +
+                "  " + task + "\n" +
+                "Now you have " + taskList.getSize() + " tasks in the list.\n";
+
```

Comment:
Perhaps when wrapping, try to break before an operator? (not included as a must but can improve readability!)

Reply:
Thanks for the suggestion!